app/models/order.rb

```diff
@@ -6,7 +6,7 @@ class Order < ActiveRecord::Base
   scope :ordered,       -> { where status: 'Ordered' }
   scope :paid,          -> { where status: 'Paid' }
   scope :cancelled,     -> { where status: 'Cancelled' }
-  scope :completed,     -> { where status: 'Complete' }
+  scope :completed,     -> { where status: 'Completed' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.